### DISCUSSION TITLE
Sprint Meeting 4/10

### DISCUSSION THREAD

**Discussion by rivermd** [Workspace-Recommendation-Engine/reco_eng#29] [Announcements]
Hi all, putting this here as our meeting notes from today (@SoohyeonB please be sure to check once you are back in Madrid 😉)

After checking in with the whole team regarding not only our current to-do's, but also revising the requirements of the project and our individual academic schedules (rip to those who have multiple final deadlines creeping up this week 🪦💀) we've decided to pivot the approach posted yesterday and instead go all hands on deck for what we have now. This means:

- completing the KNN model, implementing code to address the cold start problem by having the user rate their preference on different categories from the start within the app.

- implementing the cold start solution feature to the SAR model

- creating an API to connect the frontend to the backend

- evaluate KNN vs. SAR and ensure both have the proper documentation within the code

- Fix the MLOps pipeline issue so that all tests pass

From what I'm seeing, we're making great progress so far so let's just put this energy into full force this week for the final stretch! I will also update the issues tab in our workspace shortly so that it's updated to reflect this (and that way it doesnt feel cluttered with all the subtasks that are a part of these 'bigger' tasks) and update the assigned roles as this has shifted since we last checked in for this ⭐️

**Comment by Cdbryan** [comment]
Thanks, @rivermd!

Here is a brief to-do/assignment list based on the tasks we have left to complete to have completed by Wednesday! 
- @Cdbryan cold start-rating system 
- @farahabuatiyeh hot-start rating system 
- @SoohyeonB matches page sorting based on user location
- @rivermd @thelearner411 model API 
- @rivermd MLOps pipeline tests
